src/components/sections/campaign/AllActionsPane.scss

```diff
@@ -3,4 +3,20 @@
         @include inline-button($icon: $fa-var-plus);
         float: right;
     }
+}
+.allActionsPane-oldActions {
```

Comment:
Class names should match component names in case, i.e. this should be `.AllActionsPane-oldActions) with capital initial A.